src/kata4.humanCatDogYears.js

```diff
@@ -11,6 +11,7 @@ const humanCatDogYears = (number) => {
     dogAge = 15;
     ageArray.push(dogAge)
   }
+  // review
```

Comment:
You don't need to perform `ageArray.push` inside every if/else if/else block here...do them afterwards before your return to DRY up the code.